lib/blueprinter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Blueprinter
-  VERSION = '0.20.0'.freeze
+  VERSION = '0.20.1'.freeze
```

Comment:
This is new feature. What do you think if we make it 0.21.0?

Also need update to CHANGELOG

Reply:
Of course, great catch!